CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.12.0 (2020-04-01)
+
+- Updated the SKY UX icon font to 3.0.0-beta.1. [#105](https://github.com/blackbaud/skyux-theme/pull/105)
```

Comment:
Is `font` the right word here? It seems like it should be `version`? I might just not be following, but updating the `font to` something makes me expect to see the font name that we are updating to.

Reply:
I changed it to "SKY UX icon font version" to be more explicit.

Reply:
By the way, the name of the font is "SKY UX icons" or `skyux-icons`.

Reply:
Thanks for the clarification. I wasn't sure if font was the right word at all. Clearly it is. Thanks!